pagefaultgames/pokerogue
# [Localization] Korean Font update

## What are the changes?
Korean font, `PokePT_Wansung.ttf`, has new version now.

## Why am I doing these changes?
First version of this font is something-hurry-up version.
There are some mistakes and bad shape glyphs, and I edited them.

(Full modified list is in the commit... [in my font repo](https://github.com/MrWaterT/PokePT_Wansung).
In 77 Korean glyph modifications, about 10 of them in below is used in pokerogue.)

## What did change?
At least 20 pokemon names look better, and more simillar to original game style.
(Of course moves, abilities and items have those glyphs)

Numbers have uniform width.

ASCII glyphs have more reasonable spacing.

### Screenshots/Videos
* Pokemon names in starter select ui

Left one is modified version name. Right one in old one.

![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/5d237e1b-3316-4f21-a8c8-ae133fb4d303)
푸호꼬 (Fennekin) : Separated ㅍ and ㅜ. No Skewered ㅎ

![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/f28f3bb4-8f7c-451b-8816-e401d2f86703)
두두 (Doduo) : Separated ㄷ and ㅜ

![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/55bc51b3-a2cd-4a74-8a8b-06d38d5e0bf2)
테일로 (Taillow) : Separated ㅌ and ㅔ

![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/92831ca2-7b8f-441b-bd90-3274bb77dc0b)
동미러 (Bronzor) : Separated ㄹ and ㅓ

![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/61273509-9d0c-45ba-946a-cd6430dbd4d8)
줄뮤마 (Blitzle) : Separated ㅈ and ㅜ

![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/80420f57-45aa-41ed-8f6e-ee3e3fb3507b)
나룸퍼프 (Swirlix) : Separated ㅍ and ㅓ

![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/7375c395-7497-49ab-a1af-24a80baf4c19)
꾸왁스 (Quaxly) : Distinguishable ㅇ and ㅘ


* Statistics ui

![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/2b2d88b0-9b11-4b48-bb29-6286208d8a34)
No more narrow 1.  
314 and 729 has same width.

왕(Blastoise), 즐 and 줄 can be in somewhere in the game (common glyphs).




## How to test the changes?
Set language to Korean, just play the game.
From daily ranking board, you can see more beautiful result (score is aligned properly).

## Checklist
